include/sys/zfs_ioctl.h

```diff
@@ -219,7 +219,8 @@ typedef struct dmu_replay_record {
 			uint8_t drr_flags;
 			uint32_t drr_raw_bonuslen;
 			uint64_t drr_toguid;
-			/* only nonzero for raw streams */
+			/* only (possibly) nonzero for raw streams */
+			uint64_t drr_maxblkid;
```

Comment:
we're OK with moving the location of the following fields because we're breaking send/recv compatibility anyway?  How is the send stream incompatibility detected on the receiving side?  E.g. new send stream feature flag?

Reply:
@tcaputi can you comment on this.  Since this send stream format never was part of any tagged release we have some flexibility here about about changing it.